images/libvirt-kubevirt/Dockerfile

```diff
@@ -44,4 +44,12 @@ RUN rm -f /libvirtd.sh
 COPY libvirtd.sh /libvirtd.sh
 RUN chmod a+x /libvirtd.sh
 
+RUN echo "unix_sock_group=\"libvirt\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_ro_perms=\"0777\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_rw_perms=\"0770\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_ro=\"none\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_rw=\"none\"" >> /etc/libvirt/libvirtd.conf
```

Comment:
You must never provide passwords as command line arguments or env variables as this is insecure. Libvirt has ability to use a client config file for providing passwords as described by that link rmohr provides. The Go binding has no involvement in this process - it is all handled by the underlying libvirt library that Go calls.